create-liquid-app

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+set -e
+
+# ensure that .log directory exists
+readonly log_dir="./.log"
+mkdir -p $log_dir
+
+readonly nigiri_config_path=$(realpath ./.nigiri)
+
+# start elementsd, esplora (et al)
+nigiri --datadir $nigiri_config_path start --liquid
+
+# bring in environment variables generated by nigiri
+export $(cat .nigiri/.env | xargs)
+
+readonly mint_amount=1000000
+mint_address=$(nigiri --datadir $nigiri_config_path rpc --liquid getnewaddress)
+sleep 2
+mint_response=$(nigiri --datadir $nigiri_config_path mint $mint_address $mint_amount "Tether USD" "USDt")
+mint_asset_id=$(echo $mint_response | sed -n -e 's/^.*\(asset: \)\(.*\)\( issuance_txin: .*$\)/\2/p')
+
+echo "Minted $mint_amount USDt with asset ID $mint_asset_id to elementsd wallet address $mint_address"
+
+# start bobtimus
+readonly elementsd_rpc_user="admin1"
+readonly elementsd_rpc_password="123"
+RUST_LOG=info cargo run --bin fake_bobtimus -- \
+        --elementsd http://$elementsd_rpc_user:$elementsd_rpc_password@127.0.0.1:$LIQUID_NODE_PORT \
+        --usdt $mint_asset_id > $log_dir/bobtimus 2>&1 &
+bobtimus_pid=$!
```

Comment:
I'm afraid that personally I'll need to run this inside a docker container to be able to have waves communicate with bobtimus.

Reply:
Is there anything that we can do so that you don't need to run this in a docker container?

Reply:
Maybe nothing :) , I think I tried to connect to localhost from within a container before and it worked well: https://stackoverflow.com/questions/24319662/from-inside-of-a-docker-container-how-do-i-connect-to-the-localhost-of-the-mach